modules/apps/headless-apio/blog/blog-apio-client-test/src/testIntegration/java/com/liferay/blog/apio/client/test/BlogApioTest.java

```diff
@@ -76,6 +79,29 @@ public void setUp() throws MalformedURLException {
 		);
 	}
 
+	@After
+	public void tearDown() {
+		List<String> blogHrefs = ApioClientBuilder.given(
+		).basicAuth(
+			"test@liferay.com", "test"
+		).header(
+			"Accept", "application/hal+json"
+		).when(
+		).get(
+			_blogPostsHref
+		).then(
+		).extract(
+		).path(
+			"_embedded.BlogPosting._links.self.href"
+		);
+
+		if (blogHrefs != null) {
```

Comment:
That's cool, but what happens when you have created more blogs that the ones in your page?